examples/multisig_plain/lib.rs

```diff
@@ -598,13 +587,7 @@ mod multisig_plain {
                     })
             {
                 if self.confirmations.take(&(trans_id, *owner)).is_some() {
-                    if !self.confirmation_count.contains_key(&trans_id) {
-                        self.confirmation_count.insert(trans_id, 0);
-                    }
-                    *self
-                        .confirmation_count
-                        .get_mut(&trans_id)
-                        .expect("just inserted if missing") += 1;
+                    *self.confirmation_count.entry(trans_id).or_insert(0) += 1;
```

Comment:
Same as above: Does not preserve semantics and is less "secure" by adding garbage to our storage instead of sanely panicking when things are messed up.

Reply:
See comment above, same three lines were replaced here.